src/components/pages/About/AboutUs.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import '../../../components/common/styles/AboutUs.scss';
+
+export default function AboutUs() {
+  return (
+    <div className="about-us">
```

Comment:
Nice, we need to talk to team about getting everyones information they want displayed